suite-native/app/app.config.ts

```diff
@@ -56,8 +56,14 @@ export default ({ config }: ConfigContext): ExpoConfig => {
     const appIconIos = appIconsIos[buildType];
     const appIconAndroid = appIconsAndroid[buildType];
 
-    if (!process.env.SENTRY_AUTH_TOKEN && buildType !== 'debug') {
-        throw new Error('Missing SENTRY_AUTH_TOKEN env variable');
+    if (buildType !== 'debug') {
+        if (!process.env.SENTRY_AUTH_TOKEN) {
+            throw new Error('Missing SENTRY_AUTH_TOKEN env variable');
+        }
+
+        if (!process.env.EXPO_PUBLIC_ENVIRONMENT) {
```

Comment:
When I see this, I can see this is nonsense because if it's undefined it will default to debug so condition will be never met :D

Reply:
Reverted